openmc/plotter.py

```diff
@@ -27,10 +27,37 @@
 UNITY_MT = -1
 XI_MT = -2
 
+def get_reaction_components(mt):
+    """Determine what reactions make up redundant reaction.
+
+    Parameters
+    ----------
+    mt : int
+        ENDF MT number of the reaction to find components of.
+
+    Returns
+    -------
+    mts : list of int
+        ENDF MT numbers of reactions that make up the redundant reaction and
+        have cross sections provided.
+
+    """
+    mts = []
+    if mt in openmc.data.SUM_RULES:
+        for mt_i in openmc.data.SUM_RULES[mt]:
+            mts += get_reaction_components(mt_i)
+    if mts:
+        return mts
+    else:
+        return [mt]
+
+_FLAT_SUM_RULES = {mt: get_reaction_components(mt) for mt in
+                   openmc.data.SUM_RULES}
```

Comment:
Since this is only used for the `total` cross section, do we need to loop over all the keys? Could this be used instead inside `PLOT_TYPES_MT`, e.g.
```python
PLOT_TYPES_MT = {
    'total': get_reaction_components(1),
...
}
```